src/index.scss

```diff
@@ -0,0 +1,441 @@
+/* Fontawesome icons */
```

Comment:
- A very bad file name
- Not using neither styles module nor BEM
- Copying unnecessary general styles